tox.ini

```diff
@@ -59,8 +59,9 @@ source =
 doctests = True
 exclude = .*/,build/,dist/,instance/,venv/
 hang-closing = False
-max-doc-length = 80
-max-line-length = 80
+max-doc-length = 88
+max-line-length = 88
+extend-ignore = E203
 unused-arguments-ignore-stub-functions = True
 select = C,B,B902,B950,E,E242,F,I,U100,W
 ignore = B005,E203,E262,E266,E501,I201,W503
```

Comment:
these also

Reply:
I actually don't know what those are either. I didn't put those in. I will find out what they are for when I reorganize the project gradually.